stackit/internal/services/sqlserverflex/user/resource.go

```diff
@@ -105,6 +105,9 @@ func (r *userResource) Schema(_ context.Context, _ resource.SchemaRequest, resp
 		"user_id":     "User ID.",
 		"instance_id": "ID of the SQLServer Flex instance.",
 		"project_id":  "STACKIT project ID to which the instance is associated.",
+		"username":    "Username of the SQLServer Flex instance.",
+		"roles":       "Database access levels for the user. Possible values: [\"`##STACKIT_LoginManager##\"`, \"`##STACKIT_DatabaseManager##\"`]",
```

Comment:
Sorry, this could actually be 
```suggestion
		"roles":       "Database access levels for the user. Possible values: [`##STACKIT_LoginManager##`, `##STACKIT_DatabaseManager##`]",
```

Reply:
To match other descriptions